cephbackupstorage/cephbackupstorage/cephagent.py

```
@@ -1072,6 +1109,98 @@ def cancel(self, req):
+            raise Exception('file not found')
+
+        def _get_file_size(file_path):
+            return os.path.getsize(file_path) if os.path.exists(file_path) else 0
+
+        rsp = UploadFileProgressRsp()
+        rsp.apiId = cmd.taskUuid
+        rsp.completed = task.completed
+        rsp.size = task.expectedSize
+        rsp.actualSize = task.expectedSize
+        rsp.downloadSize = task.checked_download_size()
+        rsp.lastOpTime = long(task.lastOpTime) * 1000
```

Ruff (0.14.10)

`long` may be undefined, or defined from star imports

(F405)